frontend/packages/dev-console/integration-tests/features/topology/horizonta_pod_autoscaling/Edit_HPA_Action.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Edit HPA action
+	As a user, I want to remove the HPA assigned to a workload
+ 
+Background:
+    Given user is in developer perspective
+    And user has created a deployment/deployment-config 
+    And user has assigned values to cpu and memory value in resource limit section of advanced option
+    And user is in topology
```

Comment:
You need to update these as suggested for above file

Reply:
Same explanation as above